src/value/tests.rs

```diff
@@ -227,6 +232,74 @@ fn from_c_mg_value_string() {
     assert_eq!(format!("{}", mg_value), "\'ṰⱻⱾᵀ\'");
 }
 
+#[test]
+fn from_c_mg_value_date() {
```

Comment:
consider adding one more test with `365` days. It should give you back `1971, 1, 1`

Reply:
Done!